src/main/java/frc/trigon/robot/utilities/FilesHandler.java

```diff
@@ -0,0 +1,101 @@
+package frc.trigon.robot.utilities;
+
+import edu.wpi.first.wpilibj.Filesystem;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class FilesHandler {
+    public static final String DEPLOY_PATH = Filesystem.getDeployDirectory().getPath() + "/";
```

Comment:
didn't we have an issue with permissions? please verify that all of this works, and if it doesn't add code to manipulate the permissions